zerver/views/registration.py

```diff
@@ -352,9 +353,12 @@ def create_realm(request: HttpRequest, creation_key: Optional[Text]=None) -> Htt
             return HttpResponseRedirect(reverse('send_confirm', kwargs={'email': email}))
     else:
         form = RealmCreationForm()
+
+    recaptcha_sitekey = settings.GOOGLE_RECAPTCHA_SITEKEY
     return render(request,
                   'zerver/create_realm.html',
-                  context={'form': form, 'current_url': request.get_full_path},
+                  context={'form': form, 'current_url': request.get_full_path,
+                           'google_recaptcha_sitekey': recaptcha_sitekey},
```

Comment:
Let's put this in the default context, in `zulip/zerver/context_processors.py`. That will make it easier to add to more pages later.

Reply:
Sure.